assets/images/providers/linkedin-openid.svg

```diff
@@ -0,0 +1 @@
+linkedin.svg
```

Comment:
?

Reply:
What's wrong?

Reply:
Ah, I didn't realize it was a symlink and not an actual file. What's the difference between the two providers?